dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/AbstractOracleConnector.java

```diff
@@ -200,7 +200,7 @@ static String buildUrl(ConnectorArguments arguments) {
   }
 
   private static void checkNonUriFlags(ConnectorArguments arguments) {
-    if (arguments.getHost() != null) {
+    if (arguments.getHostOrNull() != null) {
       throw extraFlagProvided(OPT_HOST);
```

Comment:
IIRC there was no exception logged in the dumper output, or did I miss something?

Reply:
I did see the exception when I tried to reproduce the issue. Dumper's behavior can have some quirks depending on where it's run. Is it possible in your setup to verify whether the exception is thrown and not printed to the output or not thrown at all?